File: csharp/SpiceDBException.cs
namespace Borkfork.SpiceDb.Embedded;

/// <summary>
///     Thrown when SpiceDB operations fail.
/// </summary>
public sealed class SpiceDbException : Exception
{
    public SpiceDbException(string message) : base(message)
    {
    }

    public SpiceDbException(string message, Exception inner) : base(message, inner)
    {
    }
}

File: java/src/main/java/com/borkfork/spicedb/embedded/SpiceDBException.java
package com.borkfork.spicedb.embedded;

/** Thrown when SpiceDB operations fail. */
public class SpiceDBException extends RuntimeException {

  public SpiceDBException(String message) {
    super(message);
  }

  public SpiceDBException(String message, Throwable cause) {
    super(message, cause);
  }
}
